ctapipe/reco/reconstructor.py

```diff
@@ -141,6 +141,18 @@ def read(cls, path, parent=None, subarray=None, **kwargs):
         Provenance().add_input_file(path, role="reconstructor")
         return instance
 
+    def set_n_jobs(self, n_jobs):
+        """
+        Set n_jobs if applicable for the reconsructor.
+        This is not just a traits option of the class, because
```

Comment:
This is an implementation detail, not documentation. It makes sense as a public API, so it is.

Also, you can do this with traitlets by using the on change mechanism of traitlets.

Reply:
Ah, nice, I wasn't aware traitlets can do that.